go.mod

```diff
@@ -24,7 +23,6 @@ require (
 
 require (
 	github.com/beorn7/perks v1.0.1 // indirect
-	github.com/cert-manager/cert-manager v1.11.5 // indirect
```

Comment:
I think the replace and the end of the file can also be removed

Reply:
good point